package.json

```diff
@@ -28,9 +28,6 @@
     "email": "ismaelga@gmail.com"
   },
   "main": "dist/react-json-editor.js",
-  "peerDependencies": {
-    "react": ">=0.12.2 <1.0.0"
-  },
```

Comment:
@ismaelga ok, can I ask why we need a peer dependency? Is it due to lumc-nested/nested-editor#54?

Do we use lumc-nested/nested-editor?

Overall I love this project by the way. Very comprehensive.